xyz

```diff
@@ -86,6 +90,7 @@ while (($# > 0)) ; do
     -b|--branch)    branch="$1"           ; shift ;;
     -i|--increment) increment="$1"        ; shift ;;
     -m|--message)   message_template="$1" ; shift ;;
+    -e|--edit)      commit_flags+=' -e'   ;;
```

Comment:
Ah. I ordered it based on my ordering in the `--help` text, which was based on grouping related options together. I guess alphabetical is nicer for being indisputable.